### PR TITLE
travis ci configuration

### PR DESCRIPTION
**Pull request by Carlosches**

### CODE REVIEW

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v10.16.0
```
Comment (manuelq12) [inline .nvmrc]: EOF

**Review by manuelq12 — CHANGES_REQUESTED** [review]

**Review by manuelq12 — APPROVED** [review]
LGTM

File: .nvmrc (hunk 1 above)
Comment (danielgalvis98) [inline .nvmrc]: Please put here the same version that you are using in your dev environment

**Review by saradrada — COMMENTED** [review]
Daniel left a comment to change the version

**Pull request merged**